app/services/idv/flows/doc_auth_flow.rb

```diff
@@ -1,9 +1,7 @@
 module Idv
   module Flows
     class DocAuthFlow < Flow::BaseFlow
-      STEPS = {
-        welcome: Idv::Steps::WelcomeStep,
-      }.freeze
+      STEPS = {}.freeze
```

Comment:
Is the FSM clean-up for the flows being handled separately, or should `DocAuthFlow` and `DocAuthController` be deleted here as well?